packages/client/src/workflow-client.ts

```diff
@@ -138,6 +140,11 @@ export interface WorkflowHandle<T extends Workflow = Workflow> extends BaseWorkf
    */
   describe(): Promise<WorkflowExecutionDescription>;
 
+  /**
+   * Download the entire workflow history
+   */
+  fetchHistory(): Promise<FetchedHistory>;
```

Comment:
Might as well have this include the ID in it. I think we can just call it `History` and have it contain `workflowId: string` and `raw: proto.temporal.api.history.v1.History`.